inventories/providers/abb/ssc600_hypervisor_standalone_example.yaml

```diff
@@ -0,0 +1,49 @@
+# Copyright (C) 2024, SFL (https://savoirfairelinux.com)
+# SPDX-License-Identifier: Apache-2.0
+
+---
+all:
+  children:
+    standalone_machine:
+      children:
+        hypervisors:
+          hosts:
+            rtei-demoHyp:
+              ansible_host: "192.168.216.73"
+              ip_addr: "{{ ansible_host }}"
+              network_interface: "enp0s20f0u7"
```

Comment:
Put comments to explain which field must be changed by the user and which one should not.